src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -543,6 +543,7 @@
             <xsd:element name="routing" type="messenger_routing" minOccurs="0" maxOccurs="unbounded" />
             <xsd:element name="transport" type="messenger_transport" minOccurs="0" maxOccurs="unbounded" />
             <xsd:element name="bus" type="messenger_bus" minOccurs="0" maxOccurs="unbounded" />
+            <xsd:element name="stop-worker-on-signals" type="messenger_stop_worker_on_signal" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
no "s" either on the tag name, and we need ->fixXmlConfig('stop_worker_on_signal')
also, no need for a complex type for this, `<xsd:element name="stop-worker-on-signal" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />` should do it (please update the PR description accordingly)